Xero.NetStandard.OAuth2/Client/ApiException.cs

```diff
@@ -33,13 +33,13 @@ public class ApiException : Exception
         /// Gets or sets Retry-After value
         /// </summary>
         /// <value>The Retry-After (Http response header Retry-After).</value>
```

Comment:
I saw you removed my added comment regarding the actual value (eg that it is measured in minutes). As a consumer of this SDK, this saves me having to look this up online. I would strongly recommend reinstating it.